APMrover2/Log.cpp

```diff
@@ -40,6 +40,28 @@ void Rover::Log_Write_Attitude()
     DataFlash.Log_Write_PID(LOG_PIDA_MSG, g2.attitude_control.get_throttle_speed_pid().get_pid_info());
 }
 
+// Write a range finder depth message
+void Rover::Log_Write_Depth()
+{
+    // only log depth on boats with working downward facing range finders
+    if (!rover.is_boat() && rangefinder.has_data_orient(ROTATION_PITCH_270)) {
```

Comment:
This seems wrong, at least in relation to the comment. This will log on boats independently of the orientation of the rangefinder. To match the comment it should be:
- `if (!rover.is_boat() || !rangefinder.has_data_orient(ROTATION_PITCH_270))`, OR
- `if (!(rover.is_boat() && rangefinder.has_data_orient(ROTATION_PITCH_270)))`

Reply:
Jeepers, how did I make that mistake?  txs.  i'll fix!